sandbox/sandbox.go

```diff
@@ -841,11 +841,11 @@ func RemoveSandbox(sandboxDir, sandbox string, runConcurrently bool) (err error,
 		execList = append(execList, concurrent.ExecutionList{Logger: nil, Priority: 0, Command: eCommand1})
 	} else {
 		if defaults.UsingDbDeployer {
-			fmt.Printf("Running %s\n", stop)
+			// TODO: improve logging
```

Comment:
Same as mentioned above. This information should be conditional to using the code as a library of in the CLI.